core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -2410,6 +2427,15 @@ public String toString() {
      * Represents the state of the upgrade activity of Jenkins core.
      */
     public final class HudsonUpgradeJob extends DownloadJob {
+
+        /**
+          * @deprecated use {@link HudsonUpgradeJob(UpdateSite site, Authentication auth)}
```

Comment:
```suggestion
          * @deprecated use {@link HudsonUpgradeJob(UpdateSite, Authentication)}
```

How is this not a build failure? Hmm…